### PR TITLE
Feature/2.list item

### PR DESCRIPTION
**Pull request by kappa3-3**

### CODE REVIEW

File: src/components/SingleTask/index.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React, {Component} from "react";
```
Comment (EmOlsz) [inline src/components/SingleTask/index.jsx]: Add spaces after and before brackets in import, fix double quotation marks to single one

File: src/components/SingleTask/style.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
+@import "../../styles/commons";
+
+.single-task__{
```
Comment (EmOlsz) [inline src/components/SingleTask/style.scss]: the '__' should be placed after ampersand (&), so in line 2

File: src/components/SingleTask/style.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    box-sizing: border-box;;
+    height: 80px;
+    padding: 15px;
+    margin: 5px 15px;
+    border: 1px solid $gray;
+    border-radius: 5px;
+    box-shadow: 2px 1px 3px 1px $gray-opacity;
+  }
+}
+.content__{
```
Comment (EmOlsz) [inline src/components/SingleTask/style.scss]: same as above

File: src/components/TaskList/index.jsx [hunk 4]
```diff
@@ -5,6 +5,7 @@ import IconButton from '@material-ui/core/IconButton';
 import AddIcon from '@material-ui/icons/Add';
 
 import './style.scss';
+import SingleTask from "../SingleTask";
```
Comment (EmOlsz) [inline src/components/TaskList/index.jsx]: style import should be the last one

File: src/components/TaskList/style.scss [hunk 5]
```diff
@@ -3,6 +3,7 @@
 .task-list__container {
   flex-grow: 0.5;
   @include element-container;
+  flex-direction: column;
```
Comment (EmOlsz) [inline src/components/TaskList/style.scss]: including mixins should be the last one in given class declaration

File: src/components/SingleTask/index.jsx (hunk 1 above)
Comment (kappa3-3) [reply src/components/SingleTask/index.jsx]: done

File: src/components/SingleTask/style.scss (hunk 2 above)
Comment (kappa3-3) [reply src/components/SingleTask/style.scss]: done

File: src/components/SingleTask/style.scss (hunk 3 above)
Comment (kappa3-3) [reply src/components/SingleTask/style.scss]: done

File: src/components/TaskList/index.jsx (hunk 4 above)
Comment (kappa3-3) [reply src/components/TaskList/index.jsx]: done

File: src/components/TaskList/style.scss (hunk 5 above)
Comment (kappa3-3) [reply src/components/TaskList/style.scss]: done

**Pull request merged**